src/baldr/graphtile.cc

```diff
@@ -514,7 +514,7 @@ iterable_t<const DirectedEdge> GraphTile::GetDirectedEdges(const GraphId& node)
     return GetDirectedEdges(nodeinfo);
   }
   throw std::runtime_error(
-      "GraphTile NodeInfo index out of bounds: " + std::to_string(node.tileid()) + "," +
+      "GraphTile NodeInfo index out of bounds 4: " + std::to_string(node.tileid()) + "," +
```

Comment:
would a line number macro be better here for differentiation? what do 4 and 5 refer to anyway?

Reply:
:+1: Line number sounds much better. 4 and 5 refers to this being the fourth and fifth version of this exception I tracked down when debugging.

Reply:
Switched to file:line-number.